scripts/editor/icons/icons.js

```diff
@@ -217,6 +218,35 @@ export const icons = {
+	paste: <svg width='20' height='20' viewBox='0 0 20 20' fill='none' xmlns='http://www.w3.org/2000/svg'><path d='M14 7V3a1 1 0 0 0-1-1H3a1 1 0 0 0-1 1v13a1 1 0 0 0 1 1h6' stroke='currentColor' strokeWidth='1.5' strokeLinejoin='round' fill='none'/><rect x='9' y='7' width='9' height='12' rx='1' stroke='currentColor' strokeWidth='1.5' strokeLinejoin='round' fill='none'/><rect x='5' y='.5' width='6' height='3' rx='.5' fill='currentColor'/><path d='M11.5 9.5h4m-4 2.5h3m-3 2.5H14' stroke='currentColor' strokeOpacity='.12' strokeWidth='1.5' strokeLinecap='round' fill='none'/></svg>,
+	wrapperConfig: <svg width='20' height='20' viewBox='0 0 20 20' fill='none' xmlns='http://www.w3.org/2000/svg'><path d='M14.85 4h-9.7A1.15 1.15 0 0 0 4 5.15v9.7c0 .635.515 1.15 1.15 1.15h9.7A1.15 1.15 0 0 0 16 14.85v-9.7A1.15 1.15 0 0 0 14.85 4z' stroke='currentColor' strokeWidth='1.5' strokeLinecap='round' fill='none'/><path d='M16 1H4a3 3 0 0 0-3 3v12a3 3 0 0 0 3 3h12a3 3 0 0 0 3-3V4a3 3 0 0 0-3-3z' stroke='currentColor' strokeOpacity='.3' strokeWidth='1.5' strokeLinecap='round' fill='none'/><path fillRule='evenodd' clipRule='evenodd' d='M11.005 6.191c-.832-.345-1.642-.148-1.967.006a.308.308 0 0 0-.174.301c.022.304-.042.64-.204.81a.436.436 0 0 1-.33.13c-.17.003-.421-.053-.774-.239a.301.301 0 0 0-.348.049c-.148.14-.367.38-.557.661-.185.273-.37.622-.4.972a.31.31 0 0 0 .176.306c.146.068.305.166.438.284.135.12.227.246.267.367a.46.46 0 0 1-.044.381c-.085.161-.271.365-.637.596a.31.31 0 0 0-.135.333c.052.217.156.551.305.86.075.155.164.312.27.447.102.132.235.262.4.337a.3.3 0 0 0 .3-.03 1.72 1.72 0 0 1 .434-.225.774.774 0 0 1 .407-.043c.096.022.19.077.269.217.085.15.157.407.162.83.002.133.088.25.214.29.207.065.54.14.892.162.34.022.752-.002 1.073-.19.1-.058.16-.17.152-.287a1.96 1.96 0 0 1 .032-.508.858.858 0 0 1 .173-.391.37.37 0 0 1 .304-.125c.16.002.404.07.753.283a.301.301 0 0 0 .353-.028c.26-.22.805-.84.937-1.62a.31.31 0 0 0-.14-.313c-.284-.176-.572-.46-.648-.74a.49.49 0 0 1 .047-.385c.08-.145.25-.323.588-.512a.309.309 0 0 0 .148-.345 3.82 3.82 0 0 0-.309-.791c-.154-.294-.376-.619-.678-.824a.301.301 0 0 0-.324-.01c-.305.182-.52.234-.665.231a.41.41 0 0 1-.306-.13c-.174-.182-.26-.532-.265-.836a.308.308 0 0 0-.189-.28zm-.976 5.003c.666 0 1.206-.551 1.206-1.23 0-.68-.54-1.232-1.206-1.232-.666 0-1.206.551-1.206 1.231s.54 1.23 1.206 1.23z' fill='currentColor'/></svg>,
+>>>>>>> Add copy, paste and wrapper config svg icons
```

Comment:
Why does this look like a forced merge-conflict? 😄

Reply:
@goranalkovic-infinum I've rebased develop back to this branch, got conflict, fixed it, finished rebasing and force pushed the changes.

Reply:
You need to remove all the ```>>>>>```parts of the conflict notification 🙈